pkgs/development/tools/pipenv/default.nix

```diff
@@ -39,7 +39,11 @@ in buildPythonApplication rec {
 
   propagatedBuildInputs = runtimeDeps;
 
-  doCheck = false;
+  doCheck = true;
+  checkPhase = ''
+    export HOME=$PWD
```

Comment:
I'd do `export HOME=$(mktemp -d)` instead so the build directory doesn't get polluted